headers/Ex1.hpp

```diff
@@ -1,65 +1,72 @@
+#include <memory> //unique_ptr
+
 template <typename T>
 class DynamicBuffer {
```

Comment:
Please as you mentioned in the discussion of the previous exercise - add methods for accessing the objects in the buffer.